### PR TITLE
[SEED-909] Use SeedServicesApiClient for 2 more classes

### PR DESCRIPTION
**Pull request by alexmuller**
These 2 classes are currently used in the HelloMama registration app so we need them to have the retry functionality available.

Using the SeedServicesApiClient as a base class gives us that for free.

### CODE REVIEW

File: seed_services_client/message_sender.py [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
+    """
 
-        if session_http is None:
-            session_http = HTTPServiceClient(
-                url=api_url, headers={'Authorization': 'Token ' + auth_token})
-        self.session = session
-        self.session_http = session_http
+    def __init__(self, auth_token, api_url, session=None, **kwargs):
+        super(MessageSenderApiClient, self).__init__(
+            auth_token, api_url, session=session, **kwargs)
```
Comment (rudigiesler) [inline seed_services_client/message_sender.py]: This new `__init__` function is doing nothing, so we can probably remove it?

File: seed_services_client/stage_based_messaging.py [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-        if session_http is None:
-            session_http = HTTPServiceClient(
-                url=api_url, headers={'Authorization': 'Token ' + auth_token})
-        self.session = session
-        self.session_http = session_http
+    def __init__(self, auth_token, api_url, session=None, session_http=None,
+                 **kwargs):
+        super(StageBasedMessagingApiClient, self).__init__(
+            auth_token, api_url, session=session, session_http=session_http,
+            **kwargs)
```
Comment (rudigiesler) [inline seed_services_client/stage_based_messaging.py]: The same with this `__init__` function not being required.

File: seed_services_client/message_sender.py (hunk 1 above)
Comment (alexmuller) [reply seed_services_client/message_sender.py]: Ah cool, I was under the impression it was required to call the `__init__` in the parent class but less code is better. Removed, thanks!

**Comment by alexmuller** [comment]
Ready for another review, also bumped the version number.

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**